docs/user_guide/02_medrecord_intro.md

```diff
@@ -0,0 +1,239 @@
+---
+```
+This group will include all the defined nodes, allowing for easier access during complex analyses. Both nodes and edges can be added to a group, with no limitations on group size. Additionally, nodes and edges can belong to multiple groups without restriction.
+
+```{image} https://raw.githubusercontent.com/limebit/medmodels-static/main/imgs/user_guide/02/02_medrecord_intro_04.png
+:class: transparent-image
+```
+
+## Saving and Loading MedRecords
+
+When building a MedRecord, you may want to save it to create a persistent version. This can be done by storing it as a RON (Rusty Object Notation) file. The MedRecord can then be reloaded, allowing you to create a new instance from the saved RON file.
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 80-81
+---
+```
+
+## Printing Overview Tables
+
+The MedModels MedRecord class is designed to efficiently handle large datasets while maintaining a standardized data structure that supports complex analysis methods. As a result, the structure within the MedRecord can become intricate and difficult to manage. To address this, MedModels offers tools to help keep track of the graph-based data. One such tool is the `print_node_overview` method:
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 60
+---
+```
+
+It will print an overview over all grouped nodes in the MedRecord.
+
+```
+------------------------------------------------------
+Nodes Group Count Attribute Info                    
+------------------------------------------------------
+Diagnoses   1     ICD       Values: I.21             
+                  ICDv      min: 10                  
+                            max: 10                  
+                            mean: 10.00              
+Medications 2     Name      Values: Insulin, Wararin 
+Patients    3     Age       min: 64                  
+                            max: 74                  
+                            mean: 70.00              
+                  Loc       Values: GER, USA         
+                  Sex       Values: F, M             
+US-Patients 2     Age       min: 72                  
+                            max: 74                  
+                            mean: 73.00              
+                  Loc       Values: USA              
+                  Sex       Values: M                
+------------------------------------------------------
+```
+As shown, we have two groups of nodes - Patients and Medications - created when adding the nodes. Additionally, there’s a group called 'US-Patients' that we created. For each group of nodes, we can view their attributes along with a brief statistical summary, such as the minimum, maximum, and mean for numeric variables.
+
+We can do the same to get an overview over edges in our MedRecord by using the `print_edge_overview` method:
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 62
```

Comment:
Big problem! we are printing the same table twice (both nodes)

Reply:
Fixed, thank you!